osf_tests/utils.py

```diff
@@ -218,39 +218,3 @@ def get_default_test_schema():
         create_schema_blocks_for_atomic_schema(test_schema)
 
     return test_schema
-<<<<<<< HEAD
-=======
-
-
-def _ensure_subscriptions(provider):
-    '''Make sure a provider's subscriptions exist.
-
-    Provider subscriptions are populated by an on_save signal when the provider is created.
-    This has led to observed race conditions and probabilistic test failures.
-    Avoid that.
-    '''
-    for subscription in provider.DEFAULT_SUBSCRIPTIONS:
-        NotificationSubscription.objects.get_or_create(
-            _id=f'{provider._id}_{subscription}',
-            event_name=subscription,
-            provider=provider
-        )
-
-def assert_notification_correctness(send_mail_mock, expected_template, expected_recipients):
-    '''Confirms that a mocked send_mail function contains the appropriate calls.'''
-    assert send_mail_mock.call_count == len(expected_recipients)
-
-    recipients = set()
-    templates = set()
-    for _, call_kwargs in send_mail_mock.call_args_list:
-        recipients.add(call_kwargs['to_addr'])
-        templates.add(call_kwargs['mail'])
-
-    assert recipients == expected_recipients
-
-    try:
-        assert templates == {expected_template}
-    except AssertionError:  # the non-static subject attributes mean we need a different comparison
-        assert {template.tpl_prefix for template in list(templates)} == {expected_template.tpl_prefix}
-        assert {template._subject for template in list(templates)} == {expected_template._subject}
->>>>>>> upstream/hotfix/25.18.1
```

Comment:
So, are these `_ensure_subscriptions` and `assert_notification_correctness` newly added ones from 25.18.1? If so, should we update them to work with NR?